services/_shared/github_rulesets_client.py

```
@@ -397,7 +410,9 @@ def detect_enforcement(
         if stored_ruleset_id is not None and rs.get("id") == stored_ruleset_id:
             grug_match = True
             break
-        elif rs.get("name", "").startswith(GRUG_RULESET_PREFIX):
+        elif any(rs.get("name", "").startswith(p) for p in (
```

Ruff (0.15.21)

[warning] Unnecessary `elif` after `break` statement

Remove unnecessary `elif`

(RET508)